app/page/send/index.tsx

```diff
@@ -31,57 +42,55 @@ const SelectToken = ({ setTokenSelected }) => {
   )
 }
 
-const SelectAddress = ({ to, setTokenSelected, onChangeTo }) => {
-  const [invalidTo, setInvalidTo] = useState<boolean>(false)
-  const account = useAccount()
+const SelectAddress = ({ onStepBack, setTxTo }) => {
+  const [inputTo, setInputTo] = useState<HexString>("")
 
   const handleToChange = async (event: ChangeEvent<HTMLInputElement>) => {
     const value = event.target.value
-    onChangeTo(value)
-    try {
-      console.log(`${ethers.getAddress(value)}`)
-      setInvalidTo(false)
-    } catch (error) {
-      console.log(`Invalid to`)
-      setInvalidTo(true)
-    }
+    setInputTo(value)
   }
   return (
     <>
-      <StepBackHeader
-        title="Select Address"
-        onStepBack={() => {
-          setTokenSelected(null)
-          navigate(Path.Send)
-        }}
-      />
-      <div className="flex">
-        <label className="flex-1">To:</label>
+      <StepBackHeader title="Select Address" onStepBack={onStepBack}>
         <input
           type="text"
           id="to"
-          value={`${to}`}
+          value={inputTo}
           onChange={handleToChange}
-          list="suggestionTo"
-          className={`border w-96 outline-none ${
-            invalidTo ? "border-red-500" : "border-gray-300"
-          }`}></input>
-        <datalist id="suggestionTo">
-          <option value={account.address}></option>
-        </datalist>
+          className="width-full border-solid border-black border-[2px] rounded-[16px] p-[16px] text-[16px]"
+          placeholder="Enter address"
+          required
+        />
+      </StepBackHeader>
+      <div className="flex flex-col py-[24px] h-[311px]">
+        <h2 className="text-[16px]">Transaction History</h2>
+        {/* TODO: Replace with actual transaction history */}
+        <div>
+          <button
+            onClick={() => {
```

Comment:
What about rendering all accounts in current wallet for now?

Reply:
[render all accounts in current wallet](https://github.com/pilagod/waallet-extension/pull/172/commits/b842d2dd0f2279184fc3706ec00c7d79b8f6769c)